src/components/app/types.ts

```diff
@@ -590,4 +590,13 @@ export interface ExternalArgoListType{
     masterFilters,
     syncListData,
     isArgoInstalled: boolean,
-}
\ No newline at end of file
+}
+export interface EditDescRequest {
+    id: number,
+    environment_name: string,
```

Comment:
Please use camelCase

Reply:
This is existing interface could impact multiple untested places. Could look into this incremently